lib/rake/proxmox.rb

```diff
@@ -222,6 +222,46 @@ def define
             end
           end
 
+          desc 'list backup jobs'
+          task 'cluster:backup:list' do
+            puts 'list_backup_jobs: '
+            proxmox.list_backup_jobs.each do |c|
+              puts " content:#{c}"
```

Comment:
The output from `puts " content:#{c}"` is very unclear and I think, it should be formatted to produce a standardized, parse-able output.